src/main/java/com/JSONpostroute/services/SumDigitsService.java

```diff
@@ -0,0 +1,11 @@
+package com.JSONpostroute.services;
+
+import org.springframework.stereotype.Service;
+
+@Service
+public class SumDigitsService {
+    public static int sumDigits(int totalOrders) {
```

Comment:
If we think about using this function elsewhere this parameter name "totalOrders" is not coherent, something like "digits" or something more comprehensive.

Reply:
src/main/java/com/JSONpostroute/utils or common or functions and just create a class.